app/src/main/res/values/strings.xml

```diff
@@ -54,4 +54,16 @@
     <string name="settings_category_backup">Backup</string>
     <string name="settings_automatic_backup">Automatic backup</string>
     <string name="settings_backup_path">Backup path</string>
+    <string name="settings_category_dashboard">Dashboard</string>
+    <string name="settings_dashboard_duration">Duration</string>
+    <string-array name="duration_entries">
```

Comment:
This doesn't have to be in this PR, but I'd like to change the durations available as well - was thinking:
- `All time`
- `Last year` (365 days)
- `Last month` (30 days)
- `Last two weeks` (14 days)
- `Last week` (7 days)

Reply:
No objections, I think I just took the 3 durations from the Strava app :)